locales/fr_FR.json

```diff
@@ -1401,47 +1401,47 @@
           }
         },
         "toggle-limit": "Set a price for this trade",
-        "above-limit": "{{percent}}% above market",
-        "below-limit": "{{percent}}% below market",
-        "reset": "Reset",
+        "above-limit": "{{percent}}% au-dessus du marché",
+        "below-limit": "{{percent}}% en dessous du marché",
+        "reset": "Réinitialiser",
         "submit": "Review trade"
       },
       "review-order": {
-        "header": "Review your order",
+        "header": "Vérifiez votre ordre",
         "columns": {
-          "market-price": "Price per {{currencyBuy}}",
-          "limit-price": "Set price per {{currencyBuy}}",
-          "fee": "Maximum trading fee",
-          "fiat-total": "{{currencyFiat}} estimate"
-        },
-        "desc": "Your order will remain on the book until it is filled, you cancel it, or 24 hours have passed.",
-        "submit": "Create order",
-        "submit-placing": "Placing order…"
+          "prix du marché ":" Prix par {{currencyBuy}}",
+          "prix limite ":" Définir le prix par {{currencyBuy}}",
+          "fee": "Frais de négociation maximum",
```

Comment:
@BILLYBELT Are you using Google Translate as a basis? If so, could you please only input one string at a time, rather than copy/paste large groups of lines? I'm presuming this is why some keys are being translated.